detection_rules/devtools.py

```diff
@@ -66,11 +67,15 @@ def build_release(config_file, update_version_lock, release=None, verbose=True):
     if verbose:
         click.echo('[+] Building package {}'.format(config.get('name')))
 
-    package = Package.from_config(config, update_version_lock=update_version_lock, verbose=verbose)
+    package = Package.from_config(config, verbose=verbose)
+
+    if update_version_lock:
+        manage_versions(package.rules, save_changes=True, verbose=verbose)
```

Comment:
this means `manage_versions` will run twice, since it is already running at Package init

Reply:
correct, but it only saves changes and logs once

Reply:
i think calling it within the package code is pretty messy to start with.
so this seems like a step in the right direction to me, just not complete

Reply:
I agree, but at the moment, there are multiple places that rely on knowing whether a rule is new, changed, or deprecated, so as you stated, it would have to take that into account when breaking it out.

I just wanted to make sure there was nothing that would incorrectly change from it running multiple times